website/src/ci.md

```diff
@@ -8,10 +8,24 @@ layout: layouts/documentation.njk
 
 You may desire to run `dprint check` as a step on your CI to ensure the code is formatted.
 
-## GitHub Action
+## GitHub Actions
 
 See `dprint/check`: https://github.com/marketplace/actions/dprint-check-action
 
+### Caching dprint's incremental cache
+
+You can get really fast formatting times on the CI by caching the `~/.cache/dprint` folder between runs on a Linux runner.
+
+```yml
+- uses: actions/cache@v3
+  with:
+    path: |
+      ~/.cache/dprint
+    key: ${{ runner.os }}-dprint-${{ hashFiles('**/dprint.json') }}
```

Comment:
Do we need to specify the dprint version in this cache key?

In my experience, dprint `0.45.1` does not run the https://github.com/dprint/dprint/pull/858  compiled WASM plugin. But it works in `0.47.2`.

Reply:
Ah, sorry for the noise. It will not be a cache problem.
In that case, I should bump the `dprint.json` to update the plugin versions. 🙇‍♂️